From 155d33af7eeef19909d77aa0996c1d2dd8f1fa76 Mon Sep 17 00:00:00 2001
From: Palmer Dabbelt <palmer@sifive.com>
Date: Fri, 1 Dec 2017 11:32:00 -0800
Subject: [PATCH 01/49] audit: Avoid build failures on systems without renameat

renameat has been deprecated in favor of renameat2 for new ports.  This
allows the audit tests to build on RISC-V.

Reviewed-by: Christoph Hellwig <hch@lst.de>
Acked-by: Olof Johansson <olof@lixom.net>
Signed-off-by: Palmer Dabbelt <palmer@sifive.com>
---
 include/asm-generic/audit_dir_write.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/asm-generic/audit_dir_write.h b/include/asm-generic/audit_dir_write.h
index da09fb9..dd5a9dd 100644
--- a/include/asm-generic/audit_dir_write.h
+++ b/include/asm-generic/audit_dir_write.h
@@ -27,7 +27,9 @@ __NR_mknod,
 __NR_mkdirat,
 __NR_mknodat,
 __NR_unlinkat,
+#ifdef __NR_renameat
 __NR_renameat,
+#endif
 __NR_linkat,
 __NR_symlinkat,
 #endif
-- 
2.7.4

